code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -157,7 +157,9 @@
 	..()
 
 /datum/chemical_reaction/slime/slimebork/proc/getbork()
-	return get_random_food()
+	var/obj/item/food/fuud = get_random_food()
+	fuud.food_flags |= FOOD_SILVER_SPAWNED
+	return fuud
```

Comment:
Also is this tested? I think `get_random_food()` returns a typepath not a type but i'm not sure